param/__init__.py

```diff
@@ -956,25 +964,30 @@ class Boolean(Parameter):
 
     __slots__ = ['bounds']
 
-    # CB: bounds have no effect; see https://github.com/ioam/param/issues/82
-    def __init__(self,default=False,bounds=(0,1),**params):
+    # CB: bounds have no effect; see https://github.com/holoviz/param/issues/82
+    def __init__(self, default=False, bounds=(0,1), **params):
         self.bounds = bounds
-        super(Boolean, self).__init__(default=default,**params)
+        super(Boolean, self).__init__(default=default, **params)
 
     def _validate(self, val):
         if self.allow_None:
-            if not isinstance(val,bool) and val is not None:
-                raise ValueError("Boolean '%s' only takes a Boolean value or None."
-                                 %self.name)
+            if not isinstance(val, bool) and val is not None:
+                raise ValueError("Boolean '%s' only takes a Boolean value "
+                                 "or None, not type %s."
+                                 % (self.name, type(val)))
 
             if val is not True and val is not False and val is not None:
```

Comment:
@jbednar This seems like dead code, am I wrong? The `not isinstance(val, bool) and val is not None` should catch this already, right?

Reply:
I agree. There does not seem to be any possible value of `val` for which this second clause would fire that would not already have fired for the first one.